R/tidy-quote.R

```diff
@@ -262,9 +262,13 @@ expr <- function(expr) {
 #'   name. If an integer, it is passed to the `width` argument of
 #'   `expr_text()`, if `TRUE`, the default width is used. See
 #'   [exprs_auto_name()].
-#' @return A list of quosures with class `quosures`.
-#' @name quosures
+#' @param .ignore_empty Whether to ignore empty arguments. If `TRUE`,
```

Comment:
I think I'd prefer either `c("trailing", "all", "none")` or to use `NA` for the default trailing removal